README.md

```diff
@@ -64,6 +64,34 @@ end
 
 `force_encoding` is preferred to avoid JSON parsing issue when Cohere returns emoticon.
 
+`chat` supports Tool use (function calling).
+
+```ruby
+tools = [
+   {
+     name: "query_daily_sales_report",
+     description: "Connects to a database to retrieve overall sales volumes and sales information for a given day.",
+     parameter_definitions: {
```

Comment:
Why did they have to call this key differently from OpenAI and Google Gemini? 😭😭😭